src/clusterfuzz/_internal/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -3353,6 +3353,18 @@ def test_dns_resolution(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
+  def test_pysecsan_command_os_system(self):
+    """Test PySecSan command injection bug in os.system"""
+    data = self._read_test_data('pysecsan_command_os_system.txt')
+    expected_type = 'PySecSan'
+    expected_address = ''
+    expected_state = ''
```

Comment:
This should be the top three frames picked from the Python stacktrace. 

Our existing Python frame parsing regex is here: https://github.com/google/clusterfuzz/blob/2f89cd16a3cb6f2f5493c62315ac1973aac1f864/src/clusterfuzz/stacktraces/constants.py#L335